File: app/src/main/java/com/devusercode/upchat/adapter/WrapLayoutManager.java
package com.devusercode.upchat.adapter;

import android.content.Context;
import android.util.AttributeSet;

import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.devusercode.upchat.ListUsersActivity;

public class WrapLayoutManager extends LinearLayoutManager {
    public WrapLayoutManager(Context context) {
        super(context);
    }

    public WrapLayoutManager(Context context, int orientation, boolean reverseLayout) {
        super(context, orientation, reverseLayout);
    }

    @Override
    public void onLayoutChildren(RecyclerView.Recycler recycler, RecyclerView.State state) {
        try {
            super.onLayoutChildren(recycler, state);
        } catch (Exception e) {
            if (!(e instanceof IndexOutOfBoundsException)) {
                throw e;
            }
        }
    }
}

File: app/src/main/java/com/devusercode/upchat/adapter/WrapLayoutManager.kt
package com.devusercode.upchat.adapter

import android.content.Context
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.RecyclerView.Recycler

class WrapLayoutManager : LinearLayoutManager {
    constructor(context: Context?) : super(context)
    constructor(context: Context?, orientation: Int, reverseLayout: Boolean) : super(context, orientation, reverseLayout)

    override fun onLayoutChildren(
        recycler: Recycler,
        state: RecyclerView.State,
    ) {
        try {
            super.onLayoutChildren(recycler, state)
        } catch (e: Exception) {
            if (e !is IndexOutOfBoundsException) {
                throw e
            }
        }
    }
}
